man/org.freedesktop.systemd1.xml

```diff
@@ -1291,7 +1289,13 @@ node /org/freedesktop/systemd1 {
       and terminate all units that aren't dependencies of it. If <literal>ignore-dependencies</literal>, it
       will start a unit but ignore all its dependencies. If <literal>ignore-requirements</literal>, it will
       start a unit but only ignore the requirement dependencies. It is not recommended to make use of the
-      latter two options. On completion, this method returns the newly created job object.</para>
+      latter two options. On completion, if successful, this method returns the newly created job object
```

Comment:
if you touch this maybe replace the word "On completion" by "On reply" or so. i.e. to me "completion" is a bit confusing because after all the operation is *not* complete here, we return the job reference after all, which tells you about completion. By rewording this to say on "method reply" or so i think things become clearer here, because we'll just say that the method is done, not the operation. which is how it is.